snapcraft.yaml

```diff
@@ -51,7 +51,17 @@ parts:
     prime:
       - -local-debs
 
+  ubuntu-core-desktop-init:
```

Comment:
I think this part can be removed completely now, since we're launching it from the artifacts installed by the snap.

Reply:
Done.